react/week3/todos/src/components/todolist.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+
+class Form extends React.Component {
+    render() {
+      return (
+        <div>
+          <form onSubmit={this.props.handleSubmit}>
+            <label>Todo description:
+              <input value={this.props.description} onChange={this.props.handleDescriptionChange} name="description" required /><br />
+            </label>
+            <label>Deadline:
+              <input type="date" value={this.props.deadline} onChange={this.props.handleDeadlineChange} name="deadline" required /><br />
+            </label>
+            <input type='submit' value='Add todo' />
+          </form>
+        </div>
+      )
+    }
+}
+
+function FancyBorder(props) {
```

Comment:
If you want an extra challenge, you should consider looking into `styled-components` and see if you could figure out how to use this library here instead! 🤠